test/e2e/app-dir/rsc-basic.test.ts

```diff
@@ -244,32 +243,29 @@ export default function (context, { runtime, env }) {
   })
 
   it('should escape streaming data correctly', async () => {
-    const browser = await webdriver(context.appPort, '/escaping-rsc')
+    const browser = await webdriver(next.url, '/escaping-rsc')
     const manipulated = await browser.eval(`window.__manipulated_by_injection`)
     expect(manipulated).toBe(undefined)
   })
 
-  // Disable next/image for nodejs runtime temporarily
-  if (runtime === 'edge') {
-    it('should suspense next/image in server components', async () => {
-      const imageHTML = await renderViaHTTP(context.appPort, '/next-api/image')
-      const imageTag = getNodeBySelector(imageHTML, 'body > span > span > img')
+  it.skip('should suspense next/image in server components', async () => {
```

Comment:
removed the `next-api/image` test page, sth wrong with squoosh wasm? https://github.com/vercel/next.js/runs/7255088988?check_suite_focus=true

```
./public/test.jpg
TypeError: Failed to parse URL from /tmp/next-install-1657298738231/node_modules/.pnpm/file+..+next-repo-1657298738231+packages+next+next-packed.tgz_keqgimqi3cndnlhmudy42tf2su/node_modules/next/dist/server/lib/squoosh/mozjpeg/mozjpeg_node_dec.wasm
    at new Request (node:internal/deps/undici/undici:4832:19)
    at Agent.fetch2 (node:internal/deps/undici/undici:5524:29)
    at Object.fetch (node:internal/deps/undici/undici:6351:20)
    at fetch (node:internal/bootstrap/pre_execution:197:25)
    at instantiateAsync (/tmp/next-install-1657298738231/node_modules/.pnpm/file+..+next-repo-1657298738231+packages+next+next-packed.tgz_keqgimqi3cndnlhmudy42tf2su/node_modules/next/dist/server/lib/squoosh/mozjpeg/mozjpeg_node_dec.js:424:28)
    at createWasm (/tmp/next-install-1657298738231/node_modules/.pnpm/file+..+next-repo-1657298738231+packages+next+next-packed.tgz_keqgimqi3cndnlhmudy42tf2su/node_modules/next/dist/server/lib/squoosh/mozjpeg/mozjpeg_node_dec.js:447:13)
    at /tmp/next-install-1657298738231/node_modules/.pnpm/file+..+next-repo-1657298738231+packages+next+next-packed.tgz_keqgimqi3cndnlhmudy42tf2su/node_modules/next/dist/server/lib/squoosh/mozjpeg/mozjpeg_node_dec.js:1480:19
    at Object.instantiateEmscriptenWasm (/tmp/next-install-1657298738231/node_modules/.pnpm/file+..+next-repo-1657298738231+packages+next+next-packed.tgz_keqgimqi3cndnlhmudy42tf2su/node_modules/next/dist/server/lib/squoosh/emscripten-utils.js:15:12)
    at Object.dec (/tmp/next-install-1657298738231/node_modules/.pnpm/file+..+next-repo-1657298738231+packages+next+next-packed.tgz_keqgimqi3cndnlhmudy42tf2su/node_modules/next/dist/server/lib/squoosh/codecs.js:171:42)
    at Object.decodeBuffer (/tmp/next-install-1657298738231/node_modules/.pnpm/file+..+next-repo-1657298738231+packages+next+next-packed.tgz_keqgimqi3cndnlhmudy42tf2su/node_modules/next/dist/server/lib/squoosh/impl.js:52:31)
```

Reply:
Yeah there's a known issue with node 18 and squoosh https://github.com/vercel/next.js/issues/38020